js/fieldmanager-autocomplete.js

```diff
@@ -77,5 +77,9 @@ fm.autocomplete = {
 
 $( document ).ready( fm.autocomplete.enable_autocomplete );
 $( document ).on( 'fm_collapsible_toggle fm_added_element fm_displayif_toggle fm_activate_tab', fm.autocomplete.enable_autocomplete );
+$( document ).on( 'focus',
+	'input[class*="fm-autocomplete"]:not(.fm-autocomplete-enabled)',
```

Comment:
I don't think so. But the existing JS predates my involvement with FM, so I'd appreciate a review from you or someone else to confirm.